metrics/generic/generic_test.go

```diff
@@ -32,6 +33,24 @@ func TestCounter(t *testing.T) {
 	}
 }
 
+func TestIssue525(t *testing.T) {
+	counterWithoutLabels := generic.NewCounter("issue525_counter")
```

Comment:
Should this not work the same for Gauge and Histogram?

Reply:
Not sure. I know about `Gauge` and `Histogram` even less than I know about `Counter`s.